docker-compose.override.yml

```diff
@@ -59,3 +59,35 @@ services:
       - SMTP_HOST=mailcatcher
     volumes:
       - ".:/app:rw"
+  similarity_api:
+    build:
+      context: ./similarity_api
+    ports:
+      - "9100:8080"
+    env_file: .env
+    depends_on:
+      - db
+    environment:
+      - POSTGRES_HOST=postgres_docker
+    volumes:
+      - "./similarity_api/src:/app:rw"
+  owasp_zap_backend:
+    image: owasp/zap2docker-stable:latest
```

Comment:
Do we always want to pull :latest? Would there ever be a case we don't want to. Probably nothing but just figured I would mention.

Reply:
I struggle with this as it's a vulnerability scanner. So in theory you'd want to keep that as current as possible to serve as a canary for changes.  I'm not sure pinning a version will accurately capture potential issues with the code base.

Reply:
can ignore if its only the scan.

Reply:
Its also in the override, so its never deployed